types/ws/index.d.ts

```diff
@@ -346,6 +346,9 @@ declare namespace WebSocket {
         removeListener(event: string | symbol, listener: (...args: any[]) => void): this;
     }
 
+    class WebSocketServer extends Server {}
+    class WebSocket extends WebSocketAlias {}
```

Comment:
@tamuratak This seems to work well:

```suggestion
    type WebSocketServer = Server;
    type WebSocket = WebSocketAlias;
```

Are there any other issues blocking this PR from being merged?

Reply:
Thank your. It doesn't. I have added the following tests:

```ts
{
    const ws: WebSocket.WebSocket = new WebSocket.WebSocket("ws://www.host.com/path");
}

{
    const wss: WebSocket.WebSocketServer = new WebSocket.WebSocketServer({ port: 8081 });
}
```

Reply:
> Are there any other issues blocking this PR from being merged?

No, there aren't.

Reply:
I see, that's unfortunate. Those tests seem to pass with [this suggestion](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55151#discussion_r693430070), so that change could be applied.

Is there actually an issue with the way these two lines are currently defined? If not, I see no point in trying to find a more idiomatic solution to a problem that has already been solved.

Reply:
I think there is no real issue with the current pull request so far. We had better go forward with merging the pull request.

Reply:
:+1:, Unless someone can point out an issue with this approach then this should be good to merge (cc @rbuckton)